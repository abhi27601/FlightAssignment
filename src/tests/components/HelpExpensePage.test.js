import React from 'react';
import { shallow } from "enzyme";
import HelpExpensePage from '../../components/HelpExpensePage'

test('should render HelpExpensePage page', () => {
    const wrapper = shallow(<HelpExpensePage />)
    expect(wrapper).toMatchSnapshot();
})