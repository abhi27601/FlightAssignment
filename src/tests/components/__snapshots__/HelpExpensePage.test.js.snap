// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HelpExpensePage page 1`] = `
<div>
  This is from Help component.
</div>
`;
